utility/src/main/java/org/oppia/util/parser/RepositoryGlideModule.kt

```diff
@@ -1,17 +1,22 @@
 package org.oppia.util.parser
 
 import android.content.Context
+import android.graphics.drawable.PictureDrawable
 import com.bumptech.glide.Glide
 import com.bumptech.glide.Registry
 import com.bumptech.glide.annotation.GlideModule
 import com.bumptech.glide.module.AppGlideModule
+import com.caverock.androidsvg.SVG
 import java.io.InputStream
 import org.oppia.util.caching.AssetRepository
 
 /** Custom [AppGlideModule] to enable loading images from [AssetRepository] via Glide. */
 @GlideModule
 class RepositoryGlideModule : AppGlideModule() {
   override fun registerComponents(context: Context, glide: Glide, registry: Registry) {
-    registry.prepend(ImageAssetFetcher::class.java, InputStream::class.java, RepositoryModelLoader.Factory())
+    registry.register(SVG::class.java, PictureDrawable::class.java, SvgDrawableTranscoder())
+      .prepend(SVG::class.java, SvgEncoder())
```

Comment:
Do we actually need this encoder? Does Glide need it for caching the image?

Reply:
Yes we need it to write image to the stream. Otherwise image is not loaded.

Reply:
Ack thanks--I guessed it was for caching but I just wanted to be sure.